pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerMeter.java

```diff
@@ -64,7 +64,9 @@ public enum ControllerMeter implements AbstractMetrics.Meter {
   TABLE_REBALANCE_FAILURE_DETECTED("TableRebalanceFailureDetected", false),
   TABLE_REBALANCE_RETRY("TableRebalanceRetry", false),
   TABLE_REBALANCE_RETRY_TOO_MANY_TIMES("TableRebalanceRetryTooManyTimes", false),
-  NUMBER_ADHOC_TASKS_SUBMITTED("adhocTasks", false);
+  NUMBER_ADHOC_TASKS_SUBMITTED("adhocTasks", false),
+  IDEA_STATE_UPDATE_FAILURE("IdeaStateUpdateError", true),
```

Comment:
We can track it for each resource. I feel it will be more useful than tracking it globally

Reply:
(minor)
```suggestion
  IDEA_STATE_UPDATE_FAILURE("IdeaStateUpdateFailure", true),
```